README.md

```diff
@@ -178,6 +178,16 @@ upgrade_oh_my_bash
 
 Magic!
 
+## Bring Oh My Bash wherever you go through the ssh ([demo](https://asciinema.org/a/314508))
```

Comment:
Hi @anki-code,

Thanks for your hard work to support Oh My Bash in `xxh`. I really appreciate that 💯 . However, can you please move this into Oh My Bash [wiki](https://github.com/ohmybash/oh-my-bash/wiki) instead of README.md?

Cheers,
Toan

Reply:
Hi @nntoan ! What is the best wiki section for this?

Reply:
Hi @anki-code, let use https://github.com/ohmybash/oh-my-bash/wiki/External-plugins page